File: app/public_html/script/functions/samples/8.c
loop(){
    if (getSTR() >= getAGI())
        upgradeAGI(5);
    else if (getINT() >= getSTR())
        upgradeSTR(5);
    else
        upgradeINT(5);
    while(!moveTo(5,20));
    while(!moveTo(20,5));
}

File: app/public_html/script/functions/samples/8.py
def loop():
    if getSTR() >= getAGI():
        upgradeAGI(5)
    elif getINT() >= getSTR():
        upgradeSTR(5)
    else:
        upgradeINT(5)
    while not moveTo(5, 20):
        pass
    while not moveTo(20, 5):
        pass